lib/translator/common.h

```diff
@@ -27,8 +27,7 @@ namespace translator {
 
 // Vendor Identification shall be set to "NVMe" followed by 4 spaces: "NVMe    "
 // This value is not null terminated and should be size 8
-constexpr char kNvmeVendorIdentification[] = "NVMe    ";
-static_assert(strlen(kNvmeVendorIdentification) == 8);
```

Comment:
Is it possible to maintain this checks after swapping to using char*? The goal here is to prevent people from changing the length of the constant by mistake.